Docs/Build.md

```diff
@@ -2,24 +2,15 @@
 
 The information below describes the steps that the default build script (build.ps1) performs.
 
-Game configurations
--------------------
-
-The script will build the 3 supported Game configurations in Release mode:
-* Ares
-* TS
-* YR
-
 Engine configurations
 ---------------------
 
-For each Game configuration several Engine configurations will be build, depending on the platform where the script is run:
+Client use different technologies to render themselves, called Engine. Engine defines the technology used and the platform where the client can be launched:
```

Comment:
Client use -> The client uses
render themselves -> render itself

the "technology" used: this seems a little bit of weird. maybe consulting an AI to improve the text here?

Reply:
What about "The client uses different APIs to render itself, called Engine"?